### PR TITLE
[DOC-2752] Removed CDS-53849 and added a description for CDS-52846. T…

### PR DESCRIPTION
**Pull request by deepakbolangady**
…his update stems from https://harness.slack.com/archives/C013KJMDV0U/p1678281849244849. Reviewers: Sudarshan Purohit, Shylaja Sundararajan, and Naman Goenka.

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/whats-new.md [hunk 1]
```diff
@@ -16,6 +16,55 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+    delegateSelectors:
+      - harnessci-platform-ng-prod
+```
+
+Any new Jira connectors that you create must include the new `auth` object, and you must use its nested `username` and `passwordRef` fields for authentication. 
+
+The new fields override the previously used `username` and `passwordRef` authentication fields. The older fields are now deprecated.
+ 
+These changes are backward incompatible. Therefore, you must also update the Terraform provider for creating a Jira connector to the latest version (version 0.14.12) so that these new fields are provided. You also need to provide these new fields when creating Jira connectors through API.
+
```
Comment (StrawberryPad) [inline release-notes/whats-new.md:67]: "through **the** API"

**Review by StrawberryPad — DISMISSED** [review]
Just a minor missing "the"

File: release-notes/whats-new.md (hunk 1 above)
Comment (deepakbolangady) [reply release-notes/whats-new.md:67]: Thanks for the comment, Gigi. Is the article necessary when we refer to API as a general tool for information exchange? This statement does not intend to call out any specific API, such as Harness API or connector-related API.

**Review by naman-goenka — APPROVED** [review]

File: release-notes/whats-new.md (hunk 1 above)
Comment (deepakbolangady) [reply release-notes/whats-new.md:67]: > "through **the** API"

Or, will the following construction help? "You also need to provide these new fields in API calls that create or update a Jira connector.

**Pull request merged**